File: pkg/codegen/testing/utils/testdata/inline-invokes-pp/dotnet/inline-invokes.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using Aws = Pulumi.Aws;

return await Deployment.RunAsync(() => 
{
    var webSecurityGroup = new Aws.Ec2.SecurityGroup("webSecurityGroup", new()
    {
        VpcId = Aws.Ec2.GetVpc.Invoke(new()
        {
            Default = true,
        }).Apply(invoke => invoke.Id),
    });

});

File: pkg/codegen/testing/utils/testdata/inline-invokes-pp/nodejs/inline-invokes.ts
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";

const webSecurityGroup = new aws.ec2.SecurityGroup("webSecurityGroup", {vpcId: aws.ec2.getVpc({
    "default": true,
}).then(invoke => invoke.id)});
